compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/BallerinaSemanticModel.java

```diff
@@ -589,6 +590,23 @@ private void addToCompiledSymbols(Set<Symbol> compiledSymbols, Scope.ScopeEntry
         addToCompiledSymbols(compiledSymbols, scopeEntry.next, cursorPos, name, symbolEnv, states, compUnitName);
     }
 
+    private boolean checkAndUpdateModuleSymbols(Set<Symbol> compiledSymbols, Symbol evaluatingSymbol, BSymbol symbol) {
+        boolean isContaining = compiledSymbols.contains(evaluatingSymbol);
```

Comment:
```suggestion
        boolean symbolExists = compiledSymbols.contains(evaluatingSymbol);
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38093/commits/e228debdfd2b6ee4c5d3c2ee4750aedf511ea062.